src/Sarif/GrammarFiles/Sarif.g4

```diff
@@ -144,26 +144,14 @@ runInfo :
     invocationInfo?
 
     /**
-        @name {AnalysisTargets}
+        @name {FileInfo}
         @summary {
-		An array, each of whose elements is a fileReference object representing the location of 
-		a single analysis target scanned during the run. When present, this array shall contain one entry 
-		fo reach analysis target that was scanned, even if the analysis targets were not individually specified 
-		on the command line. 
-
-		NOTE 1: The command line with which the tool was invoked might specify its input files by means 
-		of a wild card such as *.cc, or it might specify them implicitly, for example, by scanning the 
-		files in the current directory.
-
-		The analysisTargets array shall be empty if no analysis targets were scanned in the course of the run. 
-
-		NOTE 2: This could happen if the command line specified a wildcard such as *.cc for the input files, 
-		and no files matched the wildcard.
+		A dictionary each of whose keys is a URI and each of whose values is an array of fileReference objects representing the location of
+		a single file target scanned during the run.
```

Comment:
the reference may be a scanned target but also could be a file against which a problem was reported (a resultFile)